paddle/scripts/docker/build.sh

```diff
@@ -49,29 +49,27 @@ cmake .. \
       -DCUDNN_ROOT=/usr/ \
       -DWITH_STYLE_CHECK=${WITH_STYLE_CHECK:-OFF} \
       -DWITH_TESTING=${WITH_TESTING:-OFF} \
-      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+      -DUNITTEST_USE_VIRTUALENV=OFF
 
 cat <<EOF
-========================================
-Building in /paddle/build ...
+============================================
+Building and installing in /paddle/build ...
```

Comment:
What is the difference between building and installing?  I think we used to build only here because the installation is postponed in building the production Docker image.  Do we have to run make install here so could we run Python test?  Or, could we run Python test without make install?